corehq/util/es/interface.py

```diff
@@ -122,6 +122,12 @@ def _fix_hits_in_results(self, results):
         for hit in hits:
             self._fix_hit(hit)
 
+        total = results['hits']['total']
```

Comment:
Is it possible that `results['hits']['total']` exists, but `results['hits']['hits']` does not? If yes, then this code will not run in that case because of the return on `KeyError` above.